tests/mmm/test_budget_optimizer.py

```diff
@@ -34,14 +51,24 @@
                     "saturation_params": {"lam": 20, "beta": 1.0},
                 },
             },
+            {
+                "method": "SLSQP",
+                "options": {"ftol": 1e-8, "maxiter": 1_002},
+            },
```

Comment:
Is there any check or sign that this is actually being used? Is there something we can check?

Reply:
Good remark! Added a specific test in https://github.com/pymc-labs/pymc-marketing/pull/737/commits/7607668bb96899d1badabd2617ec7f26d2c58550